### PR TITLE
Onboarding Documentation Updates

### PR DESCRIPTION
**Pull request by misprit7**
Added a pretty comprehensive onboarding document in preparation for new members coming to the team. Also changed some general formatting for the docs to make things easier to navigate and clearer to interpret.

### CODE REVIEW

File: doc/Onboarding.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+As for how the team works, generally work is divided between electrical, firmware and software tasks. Electrical work is things like PCB design and manufacturing, firmware is working on code that will be on the flight computer sent to space and software tasks are those pertaining to the ground station GUI. 
+
+## Github Access
+
+No matter what type of work you will be doing, you will need access to the teams Github repositories. Given this guide is itself hosted on Github you likely already know how to access the team's repos, and to get member status contact Xander. 
+
+## Next Steps
+
+Depending on what you'll primarily be working on, here are some next steps to get started contributing to the project. Even if you're only expecting to do one kind of work, if you're interested I'd definitely encourage you to set up the development environment for the other aspects of the team if only to get a fuller understanding of what's going on in the project.
```
Comment (Esouder) [inline doc/Onboarding.md:33]: "better" not "fuller"

File: doc/Onboarding.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Onboarding
+
+Welcome to Whistler-Blackcomb Avionics! We are thrilled with your new aura joining our spirit collective (this is a direct sarcastic quote from Eric).
```
Comment (Esouder) [inline doc/Onboarding.md:3]: This isn't sarcasm

File: doc/Onboarding.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# Onboarding
+
+Welcome to Whistler-Blackcomb Avionics! We are thrilled with your new aura joining our spirit collective (this is a direct sarcastic quote from Eric). 
+
+Here's a few things that you might need to know when joining the team:
```
Comment (Esouder) [inline doc/Onboarding.md:5]: might _want_ to know?

File: doc/Onboarding.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Whistler Blackcomb Project Background
+
+To get familiar with the setup and organization of the Whistler Blackcomb Project, by far the best resource is the [external design review](https://docs.google.com/document/d/1aMXL6ogCYWBh2T_MYrKgUeNxM_nnbr7gomK3zXRZ8YI/edit?usp=sharing) that was completed at the beginning of 2021. It's obviously way too long and too detailed for a full reading, but skimming over the main sections quickly will give a good sense about a high level overview of the project. For a description of what the avionics system looks like specifically, read our section in the EDR in more detail. Note that specific things might have changed since its writing, but most high level things are the same. 
+
+As for how the team works, generally work is divided between electrical, firmware and software tasks. Electrical work is things like PCB design and manufacturing, firmware is working on code that will be on the flight computer sent to space and software tasks are those pertaining to the ground station GUI. 
+
+## Github Access
```
Comment (Esouder) [inline doc/Onboarding.md]: Maybe under this 'github' heading or a separate heading, would be good to include our issue tracking and that nice project board you made a while ago?

Comment (Esouder) [inline doc/Onboarding.md]: What would you also think about a little section going over the plan for contributions, i.e. basically just "don't push your changes to master, make a pull request and request the rest of the team as reviewers".?

Comment (Esouder) [reply doc/Onboarding.md]: or maybe this is what you intend to put in that software section below.

**Review by Esouder — COMMENTED** [review]
Great! Made a couple of small comments.

**Comment by misprit7** [comment]
Just implemented the changes you suggested, I'll wait before merging this since I might want to add the software section before this Sunday.

**Comment by Esouder** [comment]
Thanks Xander, looks great!

**Pull request merged**